src/pages/RestaurantDetail/Menus/Menu.js

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { MdKeyboardArrowDown } from 'react-icons/md';
+
+export default function Menu({ category_name, foods, id }) {
+  const [isCollapsed, setIsCollapsed] = useState(false);
+  const toggleCollapsingMenu = () => {
+    setIsCollapsed(!isCollapsed);
+  };
+
+  useEffect(() => {
+    const isFirstCategory = id === 1;
+    if (isFirstCategory) {
+      setIsCollapsed(true);
+    }
+  }, [id]);
+
+  return (
+    <div className="menu">
+      <div className="menu_category" onClick={toggleCollapsingMenu}>
+        <span>{category_name}</span>
+        <MdKeyboardArrowDown
+          className={isCollapsed === true ? 'collapsed' : ''}
```

Comment:
boolean값을 이용해 아래와 같이 등식을 생략해 줄 수 있습니다.
만약 그 반대의 경우(isCollapsed가 false일 때를 조건으로 하고 싶다면)라면 어떻게 해줄 수 있을까요?!
```suggestion
          className={isCollapsed ? 'collapsed' : ''}
```

Reply:
앗 그러네요. false이면 !isCollapsed로 하면 될까용?